dev-lang/ispc/files/ispc-1.24.0-ignore-git.patch

```diff
@@ -0,0 +1,29 @@
+From: Paul Zander <negril.nx+gentoo@gmail.com>
+Don't attempt to find git
+
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -268,7 +268,7 @@ endif()
+ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/ispcrt/cmake)
+ include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/FixWindowsPath.cmake)
+ include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindLLVM.cmake)
+-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Git.cmake)
++# include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Git.cmake)
```

Comment:
Do not add commented out stuff via `diff`s, that just makes the patches larger for no gain.